### DISCUSSION TITLE
Bun vs Parcel

### DISCUSSION THREAD

**Discussion by dioptre** [parcel-bundler/parcel#7928] [General]
Anyone seen https://bun.sh/ and has compared with Parcel?

**Comment by nfhipona** [comment]
up

**Comment by dioptre** [comment]
https://bun.sh/docs/bundler

**Reply by grishy** [reply]
So, I mean maybe better to compare with only SWC (https://swc.rs/) part of Parcel. Look like you can write plugin for Parcel to add Bun (transformer).
Parcel just can transform HTML for example and etc.

**Reply by grishy** [reply]
https://github.com/swc-project/swc/discussions/2743

**Reply by grishy** [reply]
https://twitter.com/jarredsumner/status/1485397686224257027

**Comment by 81reap** [comment]
Here is an example of using Parcel with Bun
https://github.com/parcel-bundler/rsc-examples/tree/main/examples/server-bun
